src/common/models.py

```diff
@@ -3,11 +3,35 @@
 from account.models import Profile
 
 
+class Votes(models.Model):
+        upvoters = models.ManyToManyField(
+            Profile,
+            related_name="upvotes",
+        )
+        downvoters = models.ManyToManyField(
+            Profile,
+            related_name="downvotes",
+        )
+
+        def comment_item_name():
+            return self.answer_comment_item\
```

Comment:
you can use ( ) instead of \ . \ is ugly.
Like this 
```python
return (
    self.answer_comment_item
    or self.question_comment_item
    or self.comment_comment_item
)
```

or

```python
return (self.answer_comment_item
        or self.question_comment_item
        or self.comment_comment_item)
```

Reply:
Fixed in https://github.com/DevTeamSCH/vikoverflow-backend/pull/11/commits/e63e5aa44011d4d34b60f42930ed1ac657d5aa48.